src/componenets/Calculator/Calculator.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import './calculator.css';
+
+function Button(props) {
+  const { digit, className } = props;
+  return (
+    <button type="button" className={className}>
+      {digit}
+    </button>
+  );
+}
+
+Button.propTypes = {
+  digit: PropTypes.string.isRequired,
+  className: PropTypes.string,
+};
+
+Button.defaultProps = {
+  className: '',
+};
```

Comment:
- [ ] Good job creating the requested component. However, you need to move the component to a new file. You can import the component into this one and use it as intended.

Reply:
Hi @julie-ify , I did it but please can you explain why this is a problem and how this is related to the project requirements? it could be an optional suggestion?

Reply:
- [ ] @Hisoka37 it's the best practice in react to separate components for easy reading the code, debugging, and making it easier to scale in the future.

Reply:
@danmainah , Thank you so much, I appreciate that.